tls/s2n_client_cert_verify.c

```diff
@@ -27,6 +31,16 @@
 #include "utils/s2n_safety.h"
 
 
+int s2n_tls13_client_cert_verify_recv(struct s2n_connection *conn)
+{
+    /* Read the algorithm and update conn->secure.conn_sig_scheme */
+    GUARD(s2n_get_and_validate_negotiated_signature_scheme(conn, &conn->handshake.io, &conn->secure.conn_sig_scheme));
+    /* Read the rest of the signature and verify */
```

Comment:
Nit: newline. You added one to the other function in this file, and it does look cleaner.